### DISCUSSION TITLE
DAVx5 task / read only calender not listed

### DISCUSSION THREAD

**Discussion by hublex** [FossifyOrg/General-Discussion#469] [Q&A]
I do use the Kanban app deck from nextcloud. I did set my kanban boards so that they will be shown in nextcloud as calendars.
All my calendars are positive synced by DAVx5 and visible in the FossilCal except of these calendar created by Deck. In DAVx5 the calendars created by Deck are listed as task instead of calendars and as read only.

**Comment by Aga-C** [comment]
There's already a feature request for this: https://github.com/FossifyOrg/Calendar/issues/36.
